### PR TITLE
Feature/disable api key bootstrap switch

### LINKED ISSUE #1653: Disable API Key field on Proxy Backend form not working in Firefox
On the Proxy tab of the API Profile, the **Disable API Key** field does not work in Firefox (50.0a2).

screenshot from 2016-09-29 10-34-11

This prevents form submission.

### LINKED ISSUE #1656: Add i18nsupport to ProxyBackends schema and sub-schemas
The ProxyBackends schema does not have i18n support. Attach i18n to the ProxyBackends and related sub-schemas:

- apiUmbrella
  - settings

### LINKED ISSUE #1658: Remove API Wizard schema translation strings
The *Add API Wizard* was removed from our project, but translation strings still exist. Remove API Wizard related strings from en.i18n.js.

### PR DESCRIPTION
**Pull request by brylie**
Closes #1653 
Closes #1656 
Closes #1658

# Changes
- Add Bootstrap Switch toggle to *API key check* field
- Change *API key check* field label to be more semantic
- Add i18n support to ProxyBackends schema and related sub-schemas
  - apiUmbrella
    - settings
- Remove API Wizard schema i18n strings

### CODE REVIEW

File: proxy_backends/client/form/form.js [hunk 1]
```diff
@@ -11,6 +11,12 @@ import { Proxies } from '/proxies/collection';
 // NPM import
 import 'urijs';
 
+Template.proxyBackend.onRendered(() => {
+  // Make Disable API Key field easier to use
+    // Fixes browser rendering issue
+  $("[name='apiUmbrella.settings.disable_api_key']").bootstrapSwitch();
```
Comment (brylie) [inline proxy_backends/client/form/form.js]: The checkbox is not working, for some reason, in Firefox 50. The Bootstrap Switch allows the form to submit, and gives explicit text to let the user know about the field state:

screenshot from 2016-09-29 12-52-44

screenshot from 2016-09-29 12-53-39

Comment (brylie) [inline proxy_backends/client/form/form.js]: We can also refine the display of Bootstrap Switch as an enhancement.

Comment (brylie) [inline proxy_backends/client/form/form.js]: I made the button smaller.

We disagree as to whether the button 'works' in the form, literally and aesthetically.

Comment (brylie) [inline proxy_backends/client/form/form.js]: We disagree as to whether the button 'works' in the form, literally and aesthetically.

**Comment by brylie** [comment]
I made the button smaller and changed the order of the 'states' (on and off), to improve readability:

screenshot from 2016-09-29 13-03-52
screenshot from 2016-09-29 13-04-02

**Comment by brylie** [comment]
Ping @bajiat @Nazarah

**Comment by brylie** [comment]
There are a lot of [Bootstrap Switch configuration options](http://www.bootstrap-switch.org/options.html) that we can use to enhance this task at a later point.

File: proxy_backends/client/form/form.js (hunk 1 above)
Comment (nnn) [inline proxy_backends/client/form/form.js]: That looks good.

**Comment by jykae** [comment]
@brylie Reviewing

**Comment by brylie** [comment]
Kiitos @jykae

**Comment by jykae** [comment]
@brylie tested on both Chrome & Firefox, works well. Merging.

**Pull request merged**